### ISSUE TITLE
Transport / play panel unusable in Fedora Linux

### ISSUE THREAD

**Issue by Joshun** [musescore/MuseScore#21036]
### Issue type

UI bug (incorrect info or interface appearance)

### Bug description

Seems fine on Windows, but not on Fedora. I haven't tested with another distro

### Steps to reproduce

1. Open existing score or create new score
2. Observe that transport panel has tempo etc. icons overlaid on top of each other

Result is the same when transport panel is undocked
Happens regardless of font or theme, or installation method (appimage, flatpak)

### Screenshots/Screen recordings

![Screenshot from 2024-01-16 11-43-47](https://github.com/musescore/MuseScore/assets/1970509/ff74208d-382a-4674-9b90-c7c7ebc4ece3)

### MuseScore Version

4.2.0

### Regression

Yes, this used to work in MuseScore 3.x and now is broken

### Operating system

Fedora 39

**Comment by MarcSabatella** [comment]
Is this using Wayland?  It's known that a number things don't render correctly in that case.  See for instance https://github.com/musescore/MuseScore/issues/20550, which lists a number of other more specific issues.

**Comment by MarcSabatella** [comment]
Very odd.  Does "View / Restore the default layout" change anything?  Which window manager?  That looks like an extremely wide window - does anything change as you resize it?  I've seen occasional glitches like that (on Debian) immediately upon unmaximizing a window, but it never persists.

**Comment by Joshun** [comment]
> Very odd. Does "View / Restore the default layout" change anything? Which window manager? That looks like an extremely wide window - does anything change as you resize it? I've seen occasional glitches like that (on Debian) immediately upon unmaximizing a window, but it never persists.

That doesn't seem to make it disappear unfortunately, and neither does resizing.
Also, I've just tested it on my work PC running Ubuntu 22.04, with Musescore from flatpak, and the result is the same.

**Comment by AbigailBuccaneer** [comment]
This seems to be a problem unique to the Flatpak build. Reproducible on both Fedora 39 and Ubuntu 23.10. It happens when the transport controls are popped out as well. Flatpak above, AppImage below:
![image](https://github.com/musescore/MuseScore/assets/908758/b6dbdf3a-10f2-4d45-beb0-69e7ee6e2d9b)

**Comment by Joshun** [comment]
Yes after more testing looks like this is an issue with the current Flatpak build - with the latest AppImage build, this issue is not present (can't remember whether previous AppImage builds might've been affected) - I guess this can be marked as a duplicate of @AbigailBuccaneer's issue

**Comment by KelvinNovais** [comment]
Same here on Fedora 37 (although Flatpak should make it behave the same on any system)

**Comment by MarcSabatella** [comment]
See https://github.com/musescore/MuseScore/issues/21113 - it appears this might be a matter of different versions of Qt.  Flatpak should presumably be using the same one the AppImage uses (5.15.12), or else someone would need to understand better why this seems to matter.

**Comment by KelvinNovais** [comment]
@MarcSabatella yes, there are differences between versions:

AppImage:
- OS: Fedora Linux 37 (Thirty Seven), Arch.: x86_64, MuseScore version (64-bit): 4.2.1-240230938, revision: github-musescore-musescore-d757433
- **Qt version 5.15.2**

And Flatapk:
- OS: KDE Flatpak runtime, Arch.: x86_64, MuseScore version (64-bit): 4.2.0
- **Qt version 5.15.10**

As you can see Flatpak incredibly uses a greater version.

**Comment by cbjeukendrup** [comment]
The current master branch uses Qt 6.2 instead of Qt 5.15.x. It might be interesting to investigate how that influences this issue. The 4.4 release will namely be made from the master branch in a few weeks.

**Comment by cbjeukendrup** [comment]
Is this still actual in MuseScore 4.6?

**Comment by KelvinNovais** [comment]
Seems ok now:

<img width="1920" height="1048" alt="Image" src="https://github.com/user-attachments/assets/ae85dbff-40a2-4bdd-820f-cce2ed011643" />

_Note: this is the Flatpak version_

**Comment by cbjeukendrup** [comment]
Good, thanks for checking. (That would mean it was actually fixed in 4.5 already, because the FlatPak is still on 4.5 as far as I know. Anyway, doesn't really matter.)
